libpimeval/src/pimPerfEnergyTables.cpp

```diff
@@ -524,6 +525,7 @@ pimPerfEnergyTables::bitsimdPerfTable = {
       { PimCmdEnum::SUB,          { 1597,  822, 2024 } },
       { PimCmdEnum::MUL,          { 2222, 1200, 3664 } },
       { PimCmdEnum::DIV,          { 3292, 1749, 5024 } },
+      { PimCmdEnum::MUL_SCALAR,   { 1852, 1000, 3054 } },
```

Comment:
You might want to copy the numbers from line 526 here instead of line 269 which is for a different PIM variant